examples/java/PluginServer.java

```diff
@@ -0,0 +1,547 @@
+                    } else {
+                        responseObserver.onError(new Throwable("An error occured in evalScript in evaluateScript."));
+                    }
+                    responseObserver.onCompleted();
+                    logger.fine("evaluateScript completed");
+                }
+            } else {
+                logger.log(Level.WARNING, "The script request header is null.");
+                responseObserver.onError(new Throwable("The script request header is null in evaluateScript."));
+                responseObserver.onCompleted();
+            }
+            
+            return new io.grpc.stub.StreamObserver<qlik.sse.ServerSideExtension.BundledRows>() {
+                
+                @Override
+                public void onNext(ServerSideExtension.BundledRows bundledRows) {
+                    logger.fine("onNext in evaluateScript called");
+                    if(header != null) {
+                        if(header.getFunctionType()==ServerSideExtension.FunctionType.AGGREGATION) {
+                            logger.log(Level.WARNING, "Aggregation is not implemented in evaluate script.");
+                            responseObserver.onCompleted();
+                        }
+                        ServerSideExtension.BundledRows result = evalScript(header, bundledRows);
+                        if(result.getRowsCount()>0) {
+                            responseObserver.onNext(result);
+                            logger.fine("onNext in evaluateScript completed");
+                        } else {
+                            responseObserver.onError(new Throwable("An error occured in evalScript in evaluateScript."));
+                        }
+                    } else {
+                        logger.log(Level.WARNING, "The script request header is null.");
+                        responseObserver.onError(new Throwable("The script request header is null in onNext in evaluateScript."));
+                        responseObserver.onCompleted();
+                    }
+                }
+                
+                @Override
+                public void onError(Throwable t) {
+                    logger.log(Level.WARNING, "Encountered error in evaluateScript", t);
+                    responseObserver.onCompleted();
+                }
+                
+                @Override
+                public void onCompleted() {
+                    logger.fine("onCompleted in evaluateScript called");
+                    responseObserver.onCompleted();
+                    logger.fine("onCompleted in evaluateScript completed");
+                }
+            };
+        }
+        
+        private ServerSideExtension.BundledRows evalScript(ServerSideExtension.ScriptRequestHeader header) {
+            
+            logger.fine("evalScript called");
+            ServerSideExtension.BundledRows.Builder builder = ServerSideExtension.BundledRows.newBuilder();

```

Comment:
A more descriptive name for builder would be nice, that indicates what is being built when you f.e. see builder.build()